src/app/state/user/user.actions.ts

```diff
@@ -0,0 +1,21 @@
+import { createAction, props } from '@ngrx/store';
+import { User } from 'src/app/models/data-model';
+
+
+export const getCurrentUser = createAction(
```

Comment:
`getCurrentUser` and `getToken` actions look strange. Are you sure that they need parameters?